### ISSUE TITLE
kubeedge1.3.0 edge can not use local pv

### ISSUE THREAD

**Issue by guojianyu** [kubeedge/kubeedge#1875]
storageclass yaml:
![image](https://user-images.githubusercontent.com/21234963/86199184-03b17b00-bb8c-11ea-90ea-ba8a253c2fb1.png)
pv yaml:
![image](https://user-images.githubusercontent.com/21234963/86199131-e2e92580-bb8b-11ea-83dc-6d468e070f5f.png)
pvc yaml:
![image](https://user-images.githubusercontent.com/21234963/86199252-2cd20b80-bb8c-11ea-9c9a-bf74d6743482.png)
pod yaml:
![image](https://user-images.githubusercontent.com/21234963/86199281-42473580-bb8c-11ea-8bbf-9e9140698fed.png)

this is node informations:  
![image](https://user-images.githubusercontent.com/21234963/86199354-7fabc300-bb8c-11ea-959f-c980ccaa71b0.png)

**compute node that head string is node
random string is edgenode

pod  nodeselector points edgenode   It's in Pending all the time**

this is edgecore informations:
 
![image](https://user-images.githubusercontent.com/21234963/86202003-aae5e080-bb93-11ea-989c-167015a7404f.png)

**Comment by fisherxu** [comment]
@guojianyu Will you use other feature of local pv, if not, why not use hostpath to mount?

**Comment by guojianyu** [comment]
> @guojianyu Will you use other feature of local pv, if not, why not use hostpath to mount?

this is our bound need.
we solved the  problem with  other csi of local pv .
but edgecore have bug ,the pv node selector  does not match  edgenode label.
error informations:
Error: "MountVolume.NodeAffinity check failed for volume \"pvc-b93238f5-c343-4d59-afa4-ba82f6c58bc7\" (UniqueName: \"kubernetes.io/host-path/a212a8fe-b9c6-4d17-938e-35578d5a5ba5-pvc-b93238f5-c343-4d59-afa4-ba82f6c58bc7\") pod \"nginx7-2\" (UID: \"a212a8fe-b9c6-4d17-938e-35578d5a5ba5\") : No matching NodeSelectorTerms

kubeedge1.3.0  edge/pkg/edged/volume_host.go is empty method.
we completed the method

 func (evh *edgedVolumeHost) GetNodeLabels() (map[string]string, error) {
       hostname, err := os.Hostname()
       if err != nil {
              klog.Errorf("couldn't determine hostname: %v", err)
              return nil, err
       }
       if len(evh.edge.nodeName) != 0 {
              hostname = evh.edge.nodeName
       }

       return map[string]string{v1.LabelHostname:hostname}, nil

}
thank you for your help,
